compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropViewHolder.skiko.kt

```diff
@@ -140,18 +133,21 @@ internal open class InteropViewHolder(
 
         layoutNode.interopViewFactoryHolder = this
 
-        val coreModifier = platformModifier
-            .pointerInteropFilter(isInteractive = isInteractive, interopViewHolder = this)
+        val coreModifier = Modifier
             .trackInteropPlacement(this)
             .onGloballyPositioned { layoutCoordinates ->
                 layoutAccordingTo(layoutCoordinates)
                 // TODO: Should be the same as [Owner.onInteropViewLayoutChange]?
-//                container.onInteropViewLayoutChange(this)
+                // container.onInteropViewLayoutChange(this)
```

Comment:
comments?

Reply:
It's a reminder for investigation after upstream merge, ask @MatkovIvan for more details.